Charts/qtest-mgr/Chart.yaml

```
@@ -65,7 +65,7 @@ icon: https://images.g2crowd.com/uploads/product/image/social_landscape/social_l
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 
-version: 1.7.3
+version: 1.7.4
```

```suggestion
version: 1.8.0
```
I would increase minor as this contains few more features 🙂